src/gsy_e/models/strategy/forward/order_updater.py

```diff
@@ -0,0 +1,68 @@
+from dataclasses import dataclass
+
+from pendulum import duration, DateTime, Duration
+
+from gsy_e.models.market.forward import ForwardMarketSlot
+
+
+@dataclass
+class OrderUpdaterParameters:
+    """
+    Parameters of the order updater class. Includes start / end energy rate and time interval
+    between each price update.
+    """
+    update_interval: duration
+    initial_rate: float
+    final_rate: float
+    capacity_percent: float
+
+    def __post_init__(self):
+        assert 0.0 <= self.capacity_percent <= 100.0
+
+
+class OrderUpdater:
```

Comment:
Finally an understandable and easy to maintain implementation of the update of the energy rate! Thanks a lot!